docs/build/isc/v1.0.0-rc.6/docs/reference/core-contracts/accounts.md

```diff
@@ -93,24 +93,27 @@ the target chain to the sender SC's L2 account on the origin chain.
 
 :::
 
-### `foundryCreateNew(t TokenScheme) s SerialNumber`
+### `nativeTokenCreate(t TokenScheme, tn TokenName, ts TokenSymbol, td TokenDecimal) s SerialNumber`
 
-Creates a new foundry with the specified token scheme, and assigns the foundry to the sender.
+Creates a new foundry and registers it as a ERC20 and IRC30 token.
 
-You can call this end point from the CLI using `wasp-cli chain create-foundry -h`
+You can call this end point from the CLI using `wasp-cli chain create-native-token -h`
 
 #### Parameters
 
-- `t` ([`iotago::TokenScheme`](https://github.com/iotaledger/iota.go/blob/develop/token_scheme.go)): The token scheme
+- `t` ([`bytes`](https://github.com/iotaledger/iota.go/blob/develop/token_scheme.go)): The token scheme
```

Comment:
Why was this changed to `bytes`? It's still the same TokenScheme.

Reply:
Probably same reason why I didn't add the return value in the beginning. It wasn't in the above mentioned yaml 😅 

I can change it back 🤔 

```suggestion
- `t` ([`iotago::TokenScheme`](https://github.com/iotaledger/iota.go/blob/develop/token_scheme.go)): The token scheme
```